definitions/schedule.py

```diff
@@ -30,6 +30,39 @@
             Dataset("india", data_dir="subnational/india/cases"),
             Dataset("italy", data_dir="subnational/italy/cases"),
             Derivative("united-kingdom-collated",
+                       data_dir="subnational/united-kingdom/collated",
+                       dependencies=["united-kingdom",
+                                     "united-kingdom-deaths",
+                                     "united-kingdom-admissions"]),
+	        Dataset("united-kingdom-full",
+                    data_dir="subnational/united-kingdom/cases"),
+            Dataset("united-kingdom-deaths-full",
+                    data_dir="subnational/united-kingdom/deaths"),
+            Dataset("united-kingdom-admissions-full",
+                    data_dir="subnational/united-kingdom/admissions"),
+            Dataset("united-kingdom-local-full",
+                    data_dir="subnational/united-kingdom-local/cases",
+                    max_wall_clock_minutes=24*60),
```

Comment:
These all need the timeout expanding and the max wall clock increasing unless you have tested and know the full dataset will run with a 10 hour wall clock and a 6 hour thread timeout.
We had discussed increasing to something like:
timeout: 3 * 24 * 60 * 60
max_wall_clock_minutes: 7 * 24 * 60

Reply:
Added the timeout and max wall clock time. 
Also double the wall clock time in derivative. Though not sure if doubling is good enough.